### DISCUSSION TITLE
[Supabase Swift] add captcha token to sign-in with SMS OTP mothed

### DISCUSSION THREAD

**Discussion by nomadlama** [supabase/supabase#25980] [Feature Requests]
As a user, recent update for [https://github.com/supabase/supabase-swift/pull/276](add captcha token to sign-in with password methods) was very great. But what really needed is add captcha token to sign-in with SMS OTP mothed on Supabase Swift SDK. Could you please update Supabase Swift SDK to use SMS OTP with captcha?

I really appreciate all supabase team's efforts.
Thank you.

**Comment by zheldRnaskan** [comment]
agreed
